.github/workflows/release_notes_check.yml

```
@@ -0,0 +1,28 @@
+name: Release Notes Check
+
+on:
+  pull_request:
+    types:
+      # On by default if you specify no types.
+      - "opened"
+      - "reopened"
+      - "synchronize"
+      # For `skip-label` only.
+      - "labeled"
+      - "unlabeled"
+
+jobs:
+  check_release_notes:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check for a release notes update
+        # Should be used once the PR is merged:
+        # https://github.com/brettcannon/check-for-changed-files/pull/83
+        #uses: brettcannon/check-for-changed-files@v1
+        uses: marenz/check-for-changed-files@token-support-packaged
+        with:
+          token: "${{ secrets.GITHUB_TOKEN }}"
+          file-pattern: "RELEASE_NOTES.md"
+          prereq-pattern: "src/**"
+          skip-label: "skip release notes"
```

```suggestion
          skip-label: "cmd:skip-release-notes"
```

Also to make it more consistent with our label style. I updated the label already.